src/main/resources/database.sql

```diff
@@ -0,0 +1,9 @@
+CREATE SCHEMA `top_store` DEFAULT CHARACTER SET utf8 ;
+
+CREATE TABLE `top_store`.`Product` (
```

Comment:
by default, the table name and column name are lowercase or underlined
example: product, product_name, id ...